mkosi/__init__.py

```diff
@@ -2077,10 +2079,32 @@ def build_uki(
     cmd += ["build", "--linux", kimg]
     mounts += [Mount(kimg, kimg, ro=True)]
 
+    if microcodes:
+        logging.debug("microcode codepath YES")
+        # might need modification
+        initrds = initrds.copy()
+        ukify = find_binary("ukify", "/usr/lib/systemd/ukify", root=context.config.tools())
+        if ukify is None:
+            # this can't happen, we check ukify earlier
+            # but the linter doens't know and is concerned about None
+            die("ukify not found")
+        # new .ucode section support?
+        if (
+            systemd_tool_version(context.config, ukify) >= "256~devel"
+            and systemd_stub_version(context, stub) >= "256~devel"
+        ):
+            for microcode in microcodes:
+                cmd += ["--microcode", microcode]
+                mounts += [Mount(microcode, microcode, ro=True)]
+        else:
+            initrds = microcodes + initrds
```

Comment:
```suggestion
            initrds += microcodes
```

Reply:
That would change the order though, no? In this (non .ucode) codepath, the microcode is supposed to be first